### ISSUE TITLE
Codify how we manage moved content and redirects

### ISSUE THREAD

**Issue by dedemorton** [elastic/docs#1357]
Currently we have two processes:

1 - We set up redirects by submitting a ticket in the website repo: https://github.com/elastic/website-www.elastic.co. Sometimes we update the legacy redirect file in the docs repo (not sure where that went). Not all contributors are aware of this process. This process has caused problems in the past because the redirects exist indefinitely and might inadvertently affect content developed later on.

2 - We add an appendix that shows moved content with links to the new location. 

We need to settle on one approach, document it on the wiki as the way forward, and publicize it to all folks who contribute docs or review doc PRs.

It would also be good to have some kind of automated checking to let us know when content is removed (or moved) so that we can fix broken links before users encounter them.

**Comment by lcawl** [comment]
I've used method 2 (deleted pages e.g. https://www.elastic.co/guide/en/elasticsearch/reference/master/redirects.html) to enable me to pass PR link checks when content that is referenced from multiple repos moves. I've then gone back and removed the deleted pages (from the most recent branches) after the links are updated in all of the appropriate repos.  I think we therefore need a combination of methods to help us address short-term and long-term redirects.

P.S.  AFAIK the https://github.com/elastic/docs/blob/master/resources/legacy_redirects.conf file is not used at this time, but I've updated it anyway since it's easier for me to track new and old URLs there and then ask the web team to replicate the changes in the "real" list of redirects.  I agree that that process needs to be simplified and better documented.
